src/Val.h

```diff
@@ -813,11 +813,19 @@ class TableVal final : public Val, public notifier::Modifiable {
 	// given set.  Note that comparisons are done using hash keys,
 	// so errors can arise for compound sets such as sets-of-sets.
 	// See https://bro-tracker.atlassian.net/browse/BIT-1949.
-	bool EqualTo(const TableVal* v) const;
+	bool EqualTo(const TableVal& v) const;
```

Comment:
A bit clearer in API terms IMO due to null-ability implications.  Also was probably thinking const-ref in more places like this can make call sites a bit more uniform regardless of whether they've got a raw pointer or IntrusivePtr (i.e. pass in `*p` in either case instead of `p.get()` for the later).

Reply:
Sounds good to me.